biorbd_optim/mapping.py

```diff
@@ -11,19 +12,28 @@ def __init__(self, expand_idx, reduce_idx, sign_to_oppose_for_expanded=()):
 
     def expand(self, obj):
         """
-        Docstring à compléter, récupère des variables symétrisées qu'elle signe et renvoie non sym
+        Copies the elements of the input obj into an obj_expanded representing the extended list according to the mapping.
+        Example :
+       - L is an array or a casADi MX containing the elements [10, 20, 30, 40, 50, 60, 70, 80].
```

Comment:
Add e.g. before the values to show this is an example

Reply:
L is a vector numpy array or CasADi MX of generalized coordinated, velocity, acceleration or torque